pkg/admin/bootstrap.go

```diff
@@ -103,7 +103,7 @@ func RegisterOther(rt core_runtime.Runtime) error {
 		if err != nil {
 			panic(err)
 		}
-		config.AdminRegistry, err = registry.Registry(c.GetProtocol(), addrUrl) // TODO: support k8s
+		config.AdminRegistry, config.Cache, err = registry.Registry(c.GetProtocol(), addrUrl, rt.KubeClient())
```

Comment:
Here, it returns both `config.AdminRegistry` and `config.Cache`, is `config.AdminRegistry` still necessary? what do we need it for later?

Reply:
It is used at [here](https://github.com/apache/dubbo-kubernetes/blob/f89cc80d96207f70daddee1936403f243b257f3a/pkg/admin/services/generic_service_Impl.go#L35).

Reply:
If it's just used here, then I think we need to think of how to refactor it later.